src/app.ts

```diff
@@ -0,0 +1,93 @@
+import express, { Application, Request, Response, NextFunction } from "express";
+import * as bodyParser from "body-parser";
+import { check, validationResult } from "express-validator";
+
+const app: Application = express();
+
+app.use(bodyParser.json());
+
+interface todo {
+  id: number;
+  text: string;
+  complete: boolean;
+}[];
+
+let todoRepository: todo[];
+
+todoRepository = [];
```

Comment:
why not do it in oneline assign ?
```ts
let todoRepository: todo[] = [];
```

Reply:
fixed on 5e1eb54c01ef4a6b08ebc2d1c553572e5fa17e1a